examples/bilingual_document.ipynb

```diff
@@ -0,0 +1,305 @@
+      "Spanish: Estaba tumbado sobre su espalda dura, y en forma de caparazón y, al levantar un poco la cabeza, veía un vientre abombado, parduzco, dividido por partes duras en forma de arco, sobre cuya protuberancia apenas podía mantenerse el cobertor, a punto ya de resbalar al suelo.\n",
+      "----------------------------------------\n",
+      "Aligned Sentence Pairs (English -> German):\n",
+      "English: One morning, as Gregor Samsa was waking up from anxious dreams, he discovered that in bed he had been changed into a monstrous verminous bug.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "English: One morning, as Gregor Samsa was waking up from anxious dreams, he discovered that in bed he had been changed into a monstrous verminous bug.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "Aligned Sentence Pairs (Spanish -> German):\n",
+      "Spanish: Cuando Gregorio Samsa se despertó una mañana después de un sueño intranquilo, se encontró sobre su cama convertido en un monstruoso insecto.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "Spanish: Cuando Gregorio Samsa se despertó una mañana después de un sueño intranquilo, se encontró sobre su cama convertido en un monstruoso insecto.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "from fastdtw import fastdtw\n",
+    "\n",
+    "eng_spa_distance, eng_spa_alignment = fastdtw(eng_embeddings, spa_embeddings, dist=cosine)\n",
+    "eng_deu_distance, eng_deu_alignment = fastdtw(eng_embeddings, deu_embeddings, dist=cosine)\n",
+    "spa_deu_distance, spa_deu_alignment = fastdtw(spa_embeddings, deu_embeddings, dist=cosine)\n",
+    "\n",
+    "def print_aligned_sentences(alignment, sent1_list, sent2_list, lang1_name, lang2_name):\n",
```

Comment:
Is the alignment always one-to-one in the particular case of your text?
If yes, then the whole problem is not super interesting (because you don't really need DTW then). 
If no, than it would be better if you demonstrate this explicitly by also printing the sentences that were left unaligned (e.g. by displaying them as being aligned to an empty sentence on the other side), or, if you have many-to-one alignments, by finding a way to visualize this as well.

Reply:
As one of the idea: you may plot the warp path as a line over the pairwise similarity heatmap plots that I suggested above.

Reply:
also, think of using pprint